docs/node_scenarios.md

```diff
@@ -2,21 +2,22 @@
 
 The following node chaos scenarios are supported:
 
-1. **node_start_scenario**: Scenario to stop the node instance.
-2. **node_stop_scenario**: Scenario to stop the node instance.
-3. **node_stop_start_scenario**: Scenario to stop and then start the node instance.
-4. **node_termination_scenario**: Scenario to terminate the node instance.
-5. **node_reboot_scenario**: Scenario to reboot the node instance.
+1. **node_start_scenario**: Scenario to stop the node instance. node_start is the action/id when running on VMware.
```

Comment:
Think it would be better if we just updated the vmware_plugin.py and scenario file to be consistent with the wording we used currently in the node scenarios.

Reply:
Agreed. Updated the id's in vmware scenarios to match rest of the node scenarios. PTAL. Thanks.

Reply:
@chaitanyaenr should you update this documentation now?